app/src/newdocs/content/code/streams.js

```diff
@@ -0,0 +1,62 @@
+export const CreateJavascriptClient =
+`const client = new StreamrClient({
+    // See below for more options
+    auth: {
+        apiKey: 'your-api-key'
+    }
+})`
+
+export const CreateJavaClient = 'StreamrClient client = new StreamrClient(new ApiKeyAuthenticationMethod(myApiKey));'
+
+export const AuthJavascriptClient =
+`new StreamrClient({
+    auth: {
+        provider: web3.currentProvider,
+    }
+})`
+
+export const AuthJavaClient = 'StreamrClient client = new StreamrClient(new EthereumAuthenticationMethod(myEthereumPrivateKey));'
+
+export const SubscribeJavascriptClient =
+`// Subscribe to a stream
+const subscription = client.subscribe(
+    {
+        stream: 'MY-STREAM-ID',
+    },
+    function(message) {
```

Comment:
weird formatting + use arrows?

Reply:
This is copy-paste from the repo page but changed it now